### PR TITLE
add descriptions about new params for plan cache

### PR DESCRIPTION
**Pull request by lawyerphx**
### What is changed, added or deleted? (Required)

- add descriptions about new params introduced in https://github.com/pingcap/tidb/pull/16476
- fix https://github.com/pingcap/tidb/issues/16217

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: statement-summary-tables.md [hunk 1]
```diff
@@ -263,3 +263,8 @@ SQL 的基础信息：
 - `BACKOFF_TYPES`：遇到需要重试的错误时的所有错误类型及每种类型重试的次数，格式为 `类型:次数`。如有多种错误则用 `,` 分隔，例如 `txnLock:2,pdRPC:1`
 - `AVG_AFFECTED_ROWS`：平均影响行数
 - `PREV_SAMPLE_TEXT`：当 SQL 是 `COMMIT` 时，该字段为 `COMMIT` 的前一条语句；否则该字段为空字符串。当 SQL 是 `COMMIT` 时，按 digest 和 `prev_sample_text` 一起分组，即不同 `prev_sample_text` 的 `COMMIT` 也会分到不同的行
+
+和 plan cache 相关的字段
```
Comment (TomShawn) [inline statement-summary-tables.md:267]:
```suggestion
和 plan cache 相关的字段：
```

File: statement-summary-tables.md [hunk 2]
```diff
@@ -263,3 +263,8 @@ SQL 的基础信息：
...
+
+- `PLAN_CACHE_HITS`：这类 SQL 命中 plan cache 的总次数
```
Comment (TomShawn) [inline statement-summary-tables.md:269]:
```suggestion
- `PLAN_CACHE_HITS`：这类 SQL 语句命中 plan cache 的总次数。
```

File: statement-summary-tables.md [hunk 3]
```diff
@@ -263,3 +263,8 @@ SQL 的基础信息：
...
+- `PLAN_IN_CACHE`：这类 SQL 上次执行是否命中了 plan ache
```
Comment (TomShawn) [inline statement-summary-tables.md:270]:
```suggestion
- `PLAN_IN_CACHE`：这类 SQL 语句的上次执行是否命中了 plan cache。
```

File: statement-summary-tables.md (hunk 1 above)
Comment (djshow832) [inline statement-summary-tables.md:267]: This part can be merged directly into `SQL 的基础信息`. No need to add a separate paragraph for these 2 fields.

File: statement-summary-tables.md (hunk 2 above)
Comment (djshow832) [reply statement-summary-tables.md:269]: No, previous description has no period for each line. We don't need the ending period.

**Pull request merged**